data/brands/amenity/fast_food.json

```diff
@@ -908,6 +908,19 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Costco Food Court",
+      "id": "costcofoodcourt-4d2ff4",
+      "locationSet": {"include": ["us"]},
```

Comment:
Do the worldwide costcos have the food court?

Reply:
I can add a few of the countries that I found but I can find some sort of a complete list. Should I add those countries?

Reply:
Add whatever you can find and folks can build on it from there.

Reply:
I added what I could find